src/main/java/seedu/duke/ui/IntLib.java

```diff
@@ -7,4 +7,5 @@ public interface IntLib {
     int RECIPE_SUM_OF_STEPS_INDEX = 3;
     int NONEMPTY_START_NUMBER = 1;
     int NONEMPTY_START_INDEX = 0;
+    int ADD_STEP_INDEX_BREAKOUT = -23;
```

Comment:
@YatPang What is this used for? and what is the significance of `-23`?

Reply:
ui.getIndexToDelete(maxCount) only allows for entry of integers from 1 to maxCount. Users entering a negative int is not possible. Hence, since method can only return ints (no null), negative int is used to break the loop of getting a valid int for index to be deleted.